### PR TITLE
Added couponCode field to Shopping Cart

### PR DESCRIPTION
**Pull request by alexey-rise**
@olegrise please review

### CODE REVIEW

File: dist/js/common-header.js [hunk 1]
```diff
@@ -5269,6 +5272,7 @@ angular.module("risevision.common.header")
       var clearCart = function () {
         _cart.useBillToAddress = false;
         _cart.shipToAttention = "";
+        _cart.couponCode = null;
```
Comment (olegrise) [inline dist/js/common-header.js]: @alexey-rise I think it should be consistent with the code in [constructor](https://github.com/Rise-Vision/common-header/pull/575/files#diff-2ec175c12c8011573203566f05849457R5200) i.e. `_cart.couponCode = "";`

**Comment by alexey-rise** [comment]
@olegrise I redid the PR as per your notes, please review.

**Comment by olegrise** [comment]
@alexey-rise Looks good. 
@alex-deaconu you okay with this change?

**Comment by alex-deaconu** [comment]
@alexey-rise @olegrise LG :+1: . Thanks.

**Pull request merged**